addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -42,14 +43,14 @@ export function useEmojiPicker(...args) {
 
 const loadingListeners = [];
 
-export const loader = {
+export const loader = reactive({
     loadEmoji: () => loadBundle("web.assets_emoji"),
-    /** @type {{ emojiValueToShortcode: Object<string, string> }} */
+    /** @type {{ emojiValueToShortcode: Object<string, string>, shortcodesByCodepoints: {[codepoints: string]: string[], knownEmojisRegex: RegExp} }} */
     loaded: undefined,
     onEmojiLoaded(cb) {
         loadingListeners.push(cb);
     },
-};
+});
```

Comment:
Putting a `reactive()` and storing in models replaces uses of `onEmojiLoaded`.
Since this is master work, can take the opportunity to simply the code using `loader` data.

Note that the `reactive` costs more than the `onEmojiLoaded`. On local DB I see it triples the time to load emojis from mouse hover of emoji button, from 5ms to 15ms. Not that much but just wanted to point it out as one minor drawback.